support-frontend/assets/helpers/subscriptions.js

```diff
@@ -271,6 +271,18 @@ const getNewsstandSaving = (subscriptionMonthlyCost: number, newsstandWeeklyCost
 const getNewsstandPrice = (productOption: PaperProductOptions) =>
   newsstandPrices[productOption];
 
+
+// Paper product
+
+const paperHasDeliveryEnabled = (): boolean => {
+  try {
+    return window.guardian.settings.switches.experiments.paperHomeDeliveryEnabled.state === 'On';
```

Comment:
You can get this from `state.common.settings` rather than directly from the window object

Reply:
it's a bit of a hassle to pass around the state in all use cases and it locks this into redux. Given this is unchangeable through the app lifecycle I'm not sure it's worth it

Reply:
Fair enough 👍